src/hooks/twilio/useVideoRoomState.tsx

```diff
@@ -87,43 +83,25 @@ export const useVideoRoomState = (
 
     dismissVideoError();
 
-    // https://media.twiliocdn.com/sdk/js/video/releases/2.7.1/docs/global.html#ConnectOptions
-    await connect(token, {
-      name: roomName,
-      video: isActiveParticipant,
-      audio: isActiveParticipant,
-      enableDscp: true,
-    })
-      .then(setRoom)
-      .catch((error) => {
+    try {
+      joinChannel(userId, roomName, true, true);
+    } catch (error) {
+      if (error instanceof Error) {
```

Comment:
There are utility functions that might be useful with simplifying this kind of code. Snipped from `RegisterForm`:
```tsx
    } catch (e) {
      const code = errorCode(e);
      const status = errorStatus(e);
      const message = errorMessage(e);
```